### PR TITLE
Adding hack script and fixing aws.md

### PR DESCRIPTION
**Pull request by kris-nova**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1803)

### CODE REVIEW

File: hack/new-iam-user.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+fi
+
+if [ -z "$2" ]; then
+    usage
+fi
+
+GROUP=$1
+USER=$2
+
+aws iam create-group --group-name ${GROUP}
```
Comment (chrislovecnm) [inline hack/new-iam-user.sh]: Should we check that `aws` is installed?

Comment (chrislovecnm) [inline hack/new-iam-user.sh]: Also should we fail if the aws user cannot run the command?  We are not checking error exits

Comment (kris-nova) [inline hack/new-iam-user.sh]: So I added a check for the `aws` utility in `$PATH`

Unsure what you mean 

> Also should we fail if the aws user cannot run the command? We are not checking error exits

Last I checked.. all shell/bash scripts will exit if an internal command fails with a non zero exit. ..

**Comment by chrislovecnm** [comment]
waiting on e2e

**Pull request merged**